waku/v2/peermanager/peer_connector.go

```diff
@@ -176,7 +185,7 @@ func (c *PeerConnectionStrategy) shouldDialPeers(ctx context.Context) {
 			return
 		case <-ticker.C:
 			isPaused := c.isPaused()
-			_, outRelayPeers, err := c.host.Peerstore().(wps.WakuPeerstore).GroupPeersByDirection()
+			_, outRelayPeers, err := c.pm.GroupPeersByDirection()
 			if err != nil {
 				c.logger.Info("Failed to get outRelayPeers from peerstore", zap.Error(err))
```

Comment:
This was not introduced in this PR but it probably makes sense to change this to a warning
```suggestion
				c.logger.Warn("failed to get outRelayPeers from peerstore", zap.Error(err))
```